terraform/modules/ecr/README

```diff
@@ -4,11 +4,11 @@ Instructions:
 
 1. After apply terraform get the repository URL
 
-terraform output | grep repository_url
+terraform output -json ecr-module | jq .ecr | jq .repository_url
```

Comment:
Not sure if we should add another dependency? but I think we will have to refactor our code a bit anyways. So lets' get this merged for now